packages/utils/src/modern.js

```diff
@@ -15,22 +12,29 @@ export const ModernBrowsers = {
   'Mobile Safari': '10.3'
 }
 
-const modernBrowsers = Object.keys(ModernBrowsers)
-  .reduce((allBrowsers, browser) => {
-    allBrowsers[browser] = semver.coerce(ModernBrowsers[browser])
-    return allBrowsers
-  }, {})
+const modernBrowsers = new Proxy(ModernBrowsers, {
+  get (browsers, prop) {
+    if (browsers[prop] && !browsers[prop].version) {
+      const coerce = require('semver/functions/coerce')
+      browsers[prop] = coerce(browsers[prop])
+    }
+    return browsers[prop]
+  }
+})
 
 export const isModernBrowser = (ua) => {
   if (!ua) {
     return false
   }
+  const coerce = require('semver/functions/coerce')
```

Comment:
Is it ok require called per-request? (TBH no idea of perf impact when we have cache)

Reply:
This code is called in nodejs and I think require.cache is pretty fast